### ISSUE TITLE
BUG: Some of the downloaded songs skip

### ISSUE THREAD

**Issue by A1ex-N** [Footsiefat/zspotify#39]
Some of the songs i've downloaded skip a bit, almost sounds like a scratched CD. Not sure if there's anything that can be done about that.

**Comment by A1ex-N** [comment]
> Honestly others have had this problem but I have no way to fix this issue sorry.

Just a bit of extra info. The songs that skip were downloaded from a playlist, but if i download those songs by themselves they no longer skip. I've compared the hashes of the two files and they differ, so i assume maybe the songs that skip aren't actually downloading the file in whole? Not sure. The part about redownloading the file might be useful for other people to know about in case they face the skipping issue.

**Comment by Footsiefat** [comment]
Im thinking that it may be due to an issue with not leaving a rest time between streaming tracks

**Comment by Footsiefat** [comment]
Yeah thats definitley a possibility BUT there would be no easy solution for that problem

**Comment by Footsiefat** [comment]
Okay @A1ex-N I just pushing a patch which adds a small delay between downloading each song which may help since it is only having issues downloading in bulk. Other that than I probably can't do much to fix this sorry.
